### PR TITLE
UF-XYZ - Upgrade to dept44 2.x - WIP

### PR DESCRIPTION
**Pull request by jklingst**
Bumps to dept44 2.x and does a teeny tiny bit of cleaning up as well

### CODE REVIEW

File: src/test/resources/application-junit.properties [hunk 1]
```diff
@@ -0,0 +1,18 @@
+spring.main.banner-mode=off
+
+# Scheduling
+cron.expression=0 0 0 31 FEB ?
+
+# Database
+spring.datasource.url=jdbc:tc:mariadb:10.6.4:///
```
Comment (mha109) [inline src/test/resources/application-junit.properties:7]: Borde den här vara samma som ovan (någonstans). 10.6.12 dvs?

Comment (jklingst) [reply src/test/resources/application-junit.properties:7]: Unit-testprofilen ska inte ens kicka igång TC. Ska ändra det

File: src/main/java/se/sundsvall/byggrarchiver/integration/messaging/MessagingIntegration.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+        var values = Map.of(
+            "byggrCaseId", ofNullable(archiveHistory.getCaseId()).orElse(""),
+            "documentName", ofNullable(archiveHistory.getDocumentName()).orElse(""),
+            "documentType", ofNullable(archiveHistory.getDocumentType()).orElse("")
+        );
+        var htmlMessage = toBase64(replace(asString(missingExtensionHtmlTemplate), values));
+
+        var emailRequest = new EmailRequest();
+        emailRequest.setSender(sender);
```
Comment (lsjolinder) [inline src/main/java/se/sundsvall/byggrarchiver/integration/messaging/MessagingIntegration.java:57]: Builderpattern?

**Review by lsjolinder — APPROVED** [review]

File: src/main/java/se/sundsvall/byggrarchiver/integration/messaging/MessagingIntegration.java (hunk 2 above)
Comment (jklingst) [reply src/main/java/se/sundsvall/byggrarchiver/integration/messaging/MessagingIntegration.java:57]: Genererad klientkod, så nja

Comment (foobar-1337) [reply src/main/java/se/sundsvall/byggrarchiver/integration/messaging/MessagingIntegration.java:57]: Is something blocking this PR from merge?

**Comment by jklingst** [comment]
> Is something blocking this PR from merge?

It's unmerged since it is a WIP PR (altough it wasn't marked as WIP before - it's been renamed appropriately). Since it also included quite a lot of cleaning up, the idea was to have an ongoing review over time instead of dropping a massive PR with 200 changed files, as the changed file count lies at approx. 120 at the moment.

But in a bigger picture - work on it has been paused for now, since there's some discussion about rewriting most of the service.

**Pull request closed without merging**